.github/fabricbot.json

```diff
@@ -3520,6 +3520,149 @@
           }
         ]
       }
+    },
+    {
+      "taskType": "scheduled",
+      "capabilityId": "ScheduledSearch",
+      "subCapability": "ScheduledSearch",
+      "version": "1.1",
+      "config": {
+        "frequency": [
+          {
+            "weekDay": 0,
+            "hours": [
+              0,
+              6,
+              12,
+              18
+            ]
+          },
+          {
+            "weekDay": 1,
+            "hours": [
+              0,
+              6,
+              12,
+              18
+            ]
+          },
+          {
+            "weekDay": 2,
+            "hours": [
+              0,
+              6,
+              12,
+              18
+            ]
+          },
+          {
+            "weekDay": 3,
+            "hours": [
+              0,
+              6,
+              12,
+              18
+            ]
+          },
+          {
+            "weekDay": 4,
+            "hours": [
+              0,
+              6,
+              12,
+              18
+            ]
+          },
+          {
+            "weekDay": 5,
+            "hours": [
+              0,
+              6,
+              12,
+              18
+            ]
+          },
+          {
+            "weekDay": 6,
+            "hours": [
+              0,
+              6,
+              12,
+              18
+            ]
+          }
+        ],
+        "searchTerms": [
```

Comment:
Should this rule also confirm the PR has been approved❔ Not liking scenarios in which this rule runs when nobody has approved a PR, the PR owner or another collaborator runs the pipeline again, the `Unblock PRs which are no longer awaiting for a ci run` rule runs, and the PR is merged because it's "approved" and runs smoothly.

Reply:
Well, my last comment was addressing exactly this - "instead of approving the PR"

Reply:
Agreed. I hadn't seen your comment when I made mine.

Re
> instead just automate further by dropping an /azp run when the pending-ci-rerun label is removed?

That has the downside of potentially rerunning a build that just completed. We aren't telling people to remove the label but to rerun. It could work if you change the comment language in the rule where we add the label.

The other option I considered but now think is likely worse would be to "confirm the PR has been approved" in the search terms. The downside there is the tendency to merge as soon as approvals come through, leaving the bot reacting too late.

Thinking further out of the box, can the bot withdraw its rejection w/o approving the PR❔

Or, should we just label the issue, reject it, and leave it to assigned person to handle removing / ignoring the bot's rejection❔ That is, limit this change to one rule❔

Reply:
Updated the PR with this proposal